altianalysis/run_difference_with_gpao.py

```diff
@@ -27,26 +28,28 @@ def get_tile_names(folder: Path) -> List[str]:
 
 def create_one_job_one_difference(store: Store, dir_in: Path, second_dir: Path | None, input_file: str, output: Path):
     job_name = f"difference_{input_file}"
-    if second_dir is not None:
+    if second_dir:
+        secondary_file = second_dir / input_file
+        assert os.path.exists(secondary_file), "Secondary elevation does not exist to compute difference !"
```

Comment:
les assertions, normalement, c'est réservé aux tests, il vaudrait mieux avoir 
```suggestion
        if not os.path.exists(secondary_file):
          raise FileNotFoundError(f"Secondary elevation does not exist to compute difference for {input_file}")
```

Reply:
ok